frame/staking/src/pallet/mod.rs

```diff
@@ -141,6 +148,31 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxNominatorRewardedPerValidator: Get<u32>;
 
+		/// The maximum number of unapplied slashes to be stored in `UnappliedSlashes`
+		#[pallet::constant]
```

Comment:
I am not sure if there's much use in putting these into the metadata, but anyways